pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -291,7 +293,9 @@ def expected_size(self):
     @property
     def relative_path_for_content_artifact(self):
         """Return relative path."""
-        return self.data_type
+        path = self.pulp2content.pulp2_storage_path
+        metadata_file_name = os.path.basename(path)
+        return '/'.join(('repodata', metadata_file_name))
```

Comment:
How about using `os.path.join` instead?
Not critical.

Reply:
+1 to os.path.join